synapse/module_api/__init__.py

```diff
@@ -1014,9 +1015,7 @@ def looping_background_call(
                 run_as_background_process,
                 msec,
                 desc,
-                f,
-                *args,
-                **kwargs,
+                lambda: maybe_awaitable(f(*args, **kwargs)),
```

Comment:
A shame mypy couldn't have spotted this.

I wonder how feasible a mypy plugin would be here?

Reply:
`ParamSpec` would have caught this, if only mypy had support for it.
I'm not a big fan of plugins since they tie you to a specific type checker.